discovery_engine_flutter/example/.gitignore

```diff
@@ -44,3 +44,12 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
+
+# Assets related
+/assets/**/*
+
+# Keep the folder structure for assets
```

Comment:
Why we need this?

Reply:
the flutter analyzer is complaining if those directories don't exist, because they are specified in example's `pubspec.yaml` as bundled assets directories

Reply:
Can we specify only `assets/models/` in the `pubspec.yaml` and put the models in there or it will not be possible to get them out of the bundle without the explicit name? 
Having to keep this is really prone to errors when we change model

Reply:
we could put everything in `assets/models/` but it would need to be a flat structure. To bundle files from subdirectories we need to specify each directory in `pubspec.yaml` like so:

```yaml
flutter:
  assets:
    - directory/ # this includes all the files in the directory, but not in the subdirectory
    - directory/subdirectory/ # we need to additionally add this to include all the files from subdirectory
```

Reply:
Let's keep it like this then :/